packages/shared/lib/core/wallet/constants/address-type.constants.ts

```diff
@@ -1,3 +1,3 @@
 export const ADDRESS_TYPE_ED25519 = 0
-export const ADDRESS_TYPE_ALIAS = 8
+export const ADDRESS_TYPE_ACCOUNT = 8
```

Comment:
I think that instead of this, we should use the sdk enum `AddressType`

Reply:
I removed this file because I don't think it makes much sense... we can just use the SDK enum directly, wdyt?

Reply:
yeah, perfect! 👌🏼